server/fleet/hosts.go

```diff
@@ -70,73 +70,73 @@ type HostUser struct {
 type Host struct {
 	UpdateCreateTimestamps
 	HostSoftware
-	ID uint `json:"id"`
+	ID uint `json:"id" csv:"id"`
 	// OsqueryHostID is the key used in the request context that is
 	// used to retrieve host information.  It is sent from osquery and may currently be
 	// a GUID or a Host Name, but in either case, it MUST be unique
-	OsqueryHostID    string    `json:"-" db:"osquery_host_id"`
-	DetailUpdatedAt  time.Time `json:"detail_updated_at" db:"detail_updated_at"` // Time that the host details were last updated
-	LabelUpdatedAt   time.Time `json:"label_updated_at" db:"label_updated_at"`   // Time that the host labels were last updated
-	PolicyUpdatedAt  time.Time `json:"policy_updated_at" db:"policy_updated_at"` // Time that the host policies were last updated
-	LastEnrolledAt   time.Time `json:"last_enrolled_at" db:"last_enrolled_at"`   // Time that the host last enrolled
-	SeenTime         time.Time `json:"seen_time" db:"seen_time"`                 // Time that the host was last "seen"
-	RefetchRequested bool      `json:"refetch_requested" db:"refetch_requested"`
-	NodeKey          string    `json:"-" db:"node_key"`
-	Hostname         string    `json:"hostname" db:"hostname"` // there is a fulltext index on this field
-	UUID             string    `json:"uuid" db:"uuid"`         // there is a fulltext index on this field
+	OsqueryHostID    string    `json:"-" db:"osquery_host_id" csv:"-"`
+	DetailUpdatedAt  time.Time `json:"detail_updated_at" db:"detail_updated_at" csv:"detail_updated_at"` // Time that the host details were last updated
+	LabelUpdatedAt   time.Time `json:"label_updated_at" db:"label_updated_at" csv:"label_updated_at"`    // Time that the host labels were last updated
+	PolicyUpdatedAt  time.Time `json:"policy_updated_at" db:"policy_updated_at" csv:"policy_updated_at"` // Time that the host policies were last updated
+	LastEnrolledAt   time.Time `json:"last_enrolled_at" db:"last_enrolled_at" csv:"last_enrolled_at"`    // Time that the host last enrolled
+	SeenTime         time.Time `json:"seen_time" db:"seen_time" csv:"seen_time"`                         // Time that the host was last "seen"
+	RefetchRequested bool      `json:"refetch_requested" db:"refetch_requested" csv:"refetch_requested"`
```

Comment:
Do we want this (sort-of-internal to UI/backend) field on the CSV?

Reply:
Good question, Noah reviewed the list on the linked ticket and liked it, I'll leave it for now as it's not sensitive info anyway.